perfume/__init__.py

```diff
@@ -57,11 +58,17 @@ def _load(self):
         for name in dir(self):
             method = self.__getattribute__(name)
             try:
```

Comment:
Thanks for the PR.

I would prefer to keep the [EAFP](https://docs.python.org/3.4/glossary.html#term-eafp) approach here, as it would result in less operations and neater code. 

Using try-except for testing the presence of these properties does not make much sense, and a better [LBYL](https://docs.python.org/3.4/glossary.html#term-lbyl) approach would use `hasattr(method, 'perfume_args')`.

This would give the following code:
```python
try:
    self.app.route(
        method.perfume_route,
        **method.perfume_args
        )(method)
except AttributeError:
    continue            
```